python/ml4ir/base/model/optimizer.py

```diff
@@ -7,17 +7,20 @@
 def get_optimizer(
     optimizer_key: str,
     learning_rate: float,
-    learning_rate_decay: float,
-    learning_rate_decay_steps: int,
-    gradient_clip_value: float,
+    learning_rate_decay: float = 1.0,
+    learning_rate_decay_steps: int = 1000000,
+    gradient_clip_value: float = 1000000,
 ) -> tf_optimizers.Optimizer:
     # Define an exponential learning rate decay schedule
-    learning_rate_schedule = ExponentialDecay(
-        learning_rate,
-        decay_steps=learning_rate_decay_steps,
-        decay_rate=learning_rate_decay,
-        staircase=True,
-    )
+    if learning_rate_decay <= 1.0:
```

Comment:
Why do we decide on this? We also need to add a doc for this.

Reply:
Needs to be less than one to be decay. I agree with all your comments on doc.